buildSrc/src/main/kotlin/plugins/docs.gradle.kts

```diff
@@ -0,0 +1,19 @@
+import org.jetbrains.dokka.gradle.DokkaMultiModuleTask
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    id("org.jetbrains.dokka")
+}
+
+tasks {
+    withType<DokkaTask>() {
+        dokkaSourceSets.configureEach {
+            includes.from(listOf("README.md"))
```

Comment:
That `listOf` call is not really needed? I think there's an overload taking a vararg.

Reply:
Yeah, I am going to remove it